### PR TITLE
added notify user on password change, issue #546

### PR DESCRIPTION
**Pull request by woopsix**

### CODE REVIEW

File: nextcloudappstore/user/signals.py [hunk 1]
```diff
@@ -26,9 +27,20 @@ def password_changed_signal(sender, instance, **kwargs):
     :param kwargs:
     :return:
     """
+    from_mail = 'appstore@nextcloud.com'
```
Comment (BernhardPosselt) [inline nextcloudappstore/user/signals.py]: This should probably be moved into the settings/base.py

File: nextcloudappstore/user/signals.py [hunk 2]
```diff
@@ -26,9 +27,20 @@ def password_changed_signal(sender, instance, **kwargs):
...
+    mail_subect = 'Nextcloud Appstore password changed'
```
Comment (BernhardPosselt) [inline nextcloudappstore/user/signals.py]: Both messages should be translated

Comment (woopsix) [reply nextcloudappstore/user/signals.py]: since i'm not familiar with the translation, could you please provide a place so i can read about it?

Comment (BernhardPosselt) [reply nextcloudappstore/user/signals.py]: import here: https://github.com/nextcloud/appstore/blob/master/nextcloudappstore/core/models.py#L16

usage here: https://github.com/nextcloud/appstore/blob/master/nextcloudappstore/core/models.py#L60

File: nextcloudappstore/user/signals.py [hunk 3]
```diff
@@ -4,6 +4,9 @@
 from django.db.models.signals import post_save, pre_save
 from django.dispatch import receiver
 from django.core.mail import send_mail
+from django.utils.translation import gettext as _
+
+from nextcloudappstore.settings.base import NEXTCLOUD_EMAIL
```
[the remark this answers is not in the document]
Comment (woopsix) [reply nextcloudappstore/user/signals.py]: Thanks a lot for the hints.

**Comment by BernhardPosselt** [comment]
Thanks!

**Comment by BernhardPosselt** [comment]
BTW, pushed some docs on it and changed *NEXTCLOUD_FROM_EMAIL* to the already included by default *DEFAULT_FROM_EMAIL*

**Pull request merged**